CondFormats/L1TObjects/interface/CaloParams.h

```diff
@@ -128,9 +128,13 @@ namespace l1t {
       // veto region is seed tower +/- <=egIsoVetoNrTowersPhi
       unsigned isoVetoNrTowersPhi_;
 
+      // turn on/off EG ID cuts
```

Comment:
@fwyzard This is still the same Stage1 module `L1TStage1Layer2Producer`, used with `auto:run2_data_relval` conditions. I guess this is one of the RelVal workflows.